admin_guide/managing_projects.adoc

```diff
@@ -237,6 +237,15 @@ with the `*ProjectRequestLimit*`
 link:../architecture/additional_concepts/admission_controllers.html[admission
 control plug-in].
 
+[IMPORTANT]
+====
+For {product-title} 3.1 and earlier project request templates created using
+`oadm create-bootstrap-project-template -o yaml > template.yaml`, the generated
```

Comment:
I think instead of repeating the full command here, you could say "using the process described in Modifying the Template for New Projects" and link to that section.